storage/src/main/scala/coop/rchain/storage/LMDBStore.scala

```diff
@@ -0,0 +1,282 @@
+package coop.rchain.storage
+import java.nio.ByteBuffer
+
+import cats.implicits._
+import coop.rchain.models.Serialize
+import coop.rchain.storage.datamodels.{BytesList, PsKsBytes, PsKsBytesList}
+import org.lmdbjava.{Dbi, Env, Txn}
+
+class LMDBStore[C, P, A, K] private (env: Env[ByteBuffer],
+                                     _dbKeys: Dbi[ByteBuffer],
+                                     _dbPsKs: Dbi[ByteBuffer],
+                                     _dbAs: Dbi[ByteBuffer],
+                                     _dbJoins: Dbi[ByteBuffer])(implicit sc: Serialize[C],
+                                                                pc: Serialize[P],
+                                                                ac: Serialize[A],
+                                                                kc: Serialize[K],
+                                                                bl: Serialize[BytesList])
+    extends IStore[C, P, A, K] {
+
+  import coop.rchain.storage.util._
```

Comment:
I would move this import to the top of the file.  

I was suggesting that you move imports for **this class**'s companion object to this position, nothing else.